tests/inspect/inspect.go

```diff
@@ -133,6 +137,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	if globalFlags.PrintNoNewPrivs {
+		r1, _, err := syscall.Syscall(
+			syscall.SYS_PRCTL,
+			uintptr(unix.PR_GET_NO_NEW_PRIVS),
+			uintptr(0), uintptr(0),
+		)
+
+		fmt.Printf("r1: %v err: %v\n", r1, err)
```

Comment:
Could this be printed in a more human-friendly way? e.g. "no_new_privs: 1" or something